appveyor.yml

```diff
@@ -3,6 +3,9 @@ image: Visual Studio 2022
 
 version: '{build}'
 
+environment:
+  TARGET_FRAMEWORK: net472
```

Comment:
Why is this needed?

Reply:
same question

Reply:
Sure, since yesterday, we updated the PKSimRDependencyResolution.cproj file, remember?
Before this update, the folder containing the files was "src\PKSimRDependencyResolution\bin\debug\"
But now, I can see (both the pipeline and local, of course) the folder has changed to "src\PKSimRDependencyResolution\bin\debug\net472\", so this is the folder to ZIP and distribute.
as net472 is the target framework, it did nos seem like a good idea to hardcode that value, but better set it on a variable.

Reply:
ah yes, I didn't see how it was being used below. It's fine for me, and I would also have accepted it as hard-coded in the path.

In other words, I don't have a strong preference.